index.html

```diff
@@ -0,0 +1,204 @@
+              ></path>
+            </g></svg
+        ></a>
+        <div
+          class="d-flex align-items-center align-items-lg-end h-100 header-right"
+        >
+          <button
+            class="btn text-light p-0 pr-2 pl-2 mr-4 align-items-center text-uppercase d-none d-sm-block btn-1 fw-bold"
+            type="button"
+          >
+            login
+          </button>
+          <button
+            class="btn rounded-pill d-flex align-items-center text-uppercase justify-content-center btn-2"
+            type="button"
+          >
+            <b>subscribe ></b>
+          </button>
+          <a href="#" class="nav-link ham-fa"
+            ><i class="fa ham-fa fa-bars" aria-hidden="true"></i
+          ></a>
+        </div>
+      </div>
+    </header>
+
+    <nav class="navbar-expand-lg navbar-light">
+      <ul
+        class="nav bottom-nav collapse navbar-collapse text-capitalize text-dark border-bottom"
+      >
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">U.S</a>
+        </li>
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">world</a>
+        </li>
+        <li class="flex-grow-1 nav-item">
+          <a class="nav-link border-right" href="#">business</a>
+        </li>
+        <li class="flex-grow-1 nav-item">
+          <a class="nav-link border-right" href="#">tech & science</a>
+        </li>
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">culture</a>
+        </li>
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">newsgeek</a>
+        </li>
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">sports</a>
+        </li>
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">health</a>
+        </li>
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">the debate</a>
+        </li>
+        <li class="flex-grow-1 nav-item border-right">
+          <a class="nav-link" href="#">vantage</a>
+        </li>
+        <li class="flex-grow-1 nav-item">
+          <a class="nav-link" href="#">weather</a>
+        </li>
+        <li class="flex-grow-1 nav-item">
+          <input
+            type="text"
+            class="form-control form-control-sm border-0"
+            placeholder="search"
+          /><i class="fa fa-search top-fa" aria-hidden="true"></i>
```

Comment:
As a previous reviewer pointed this out, you need to surround this input with a form tag, it should be a form